src/test/java/code401challenges/linkedlist/LinkedListTest.java

```diff
@@ -118,4 +119,21 @@ public void insertAfterTest(){
         assertEquals("This is all the values in the node from beginning to end\nNode Value = 10\nNode Value = 9\nNode Value = 7\nNode Value = 6\nNode Value = 5\nNode Value = 3",
                 test.toString());
     }
+
+    @Test
+    public void kthvalTest(){
+        assertEquals(
+                7,
+                test.kthFromTheLast(3)
+        );
+
+    }
+
+//    https://www.baeldung.com/junit-assert-exception
+    @Test (expected = NullPointerException.class)
+    public void whenExceptionThrown_thenExpectationSatisfied() {
+        test.prepend(3);
+        test.kthFromTheLast(20);
+
+    }

```

Comment:
It would be a good idea to add more tests for this method. Some ideas:
- method called on first node in the list
- method called on the last node in the list
- method called on an empty list
- method called on a one node list